src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -35,6 +35,20 @@
         <span>This node is a different build of the same repository as the focused node.</span>
         </td>
       </tr>
+      <tr>
+        <td>
+        </td>
+        <td>
+          <span class='table-danger'>This node is incoherent</span>
```

Comment:
>table-danger [](start = 23, length = 12)

Color alone is insufficient for accessibility reasons. We'll need an icon or text or something in addition, which will get interesting, since it will conflict with the existing icons.  Maybe "[not-equal](https://fontawesome.com/icons/not-equal?style=solid)". It will also need a "title" of some kind.

Though maybe this icon is "less important" than the others, so we only show this icon unless some other icon is more important?

Reply:
Added the same exclamation mark next to the build number on the line, like we have for incoherent dependencies for build that rely on incoherent dependencies, and updated the alt text for those.

Reply:
Won't that be confusing? The same icon now means two things.  We really need a different thing here to identify which kind of problem it is, since, other than color, there is no indication to a user what's interesting about this row.  For color blind people, this is hard to use.  We should really use some different icon, or text, or something.  The "title" wont really be visible to people, because when your mouse gets near there, it turns into the "lock" icon.